### DISCUSSION TITLE
Captcha

### DISCUSSION THREAD

**Discussion by ctkubik** [jdholtz/auto-southwest-check-in#64] [General]
This is a bit of a random thing...ever since Chrome 110 (now 111) I get the "are you human" captchas on every single site I go to.  I have this utility running all the time...could that be the reason I'm getting hammered with the captcha stuff? Is anyone else seeing this issue?

**Comment by jdholtz** [comment]
This could potentially be due to the User agent set in this script. The sites could be reading cookies from the script's browser session and seeing that the User agent doesn't match with your OS (although this is just a guess). The user agent will be removed in the next version (see #58), so you can see if you still get this issue after that. 

Alternatively, you could use the Docker image to fully isolate your Chrome session from this script's session. If it still happens, the script is not the issue then.
